mods/pageviews.js

```diff
@@ -320,9 +305,25 @@ PJVS.prototype.pageviewsForProjects = function(restbase, req) {
             }
         }
 
-    });
+    }).catch(notFoundCatcher);
 
-    return dataRequest.then(normalizeResponse);
+    return dataRequest.then(normalizeResponse).then(function(res) {
+        if (res.body.items) {
+            res.body.items.forEach(function(item) {
+                // prefer the v column if it's loaded
+                if (item.hasOwnProperty('v') && item.v !== null) {
```

Comment:
I mean I could do `if ('v' in item)`, but eslint doesn't love that in browsers so I don't usually use it.  Since `if (item.v)` is not correct what else would you like to see here.  Also, what's wrong with `hasOwnProperty`, this is exactly what it was meant for.

As a sort of higher level point, I don't think it's too productive to discuss tiny details like this in code review.  If this introduced a problem with the code you could suggest a test that would fail because of it.  And I'd be happy to implement the test and the fix.  But if it works, we waste time by explaining to each other style nuances and correctness nuances that we might not have thought of.